### DISCUSSION TITLE
Change the .venv folder

### DISCUSSION THREAD

**Discussion by va1ker** [python-poetry/poetry#8309] [Q&A]
Hello, I'm stuck with problem in Poetry and when i have this in `Poetry.toml`:
```
[virtualenvs]
create = true
in-project = true
```
`Poetry shell` will creaete a `.venv` folder in site of project. Is there any way to change the name of the `.venv` folder to `venv`?

**Comment by dimbleby** [comment]
no

**Reply by va1ker** [reply]
Why? Can you please throw documentation link for that?
